index.html

```diff
@@ -52,13 +52,31 @@ <h1 class="header-title">
         <div class="application-demo__container">
           <img
             class="application-demo__img"
-            src="/images/Group 53 (1).png"
+            src="images/Group 53 (1).png"
             alt="application demo"
           />
           <hr class="application-demo__line" />
         </div>
       </div>
     </section>
-    <section>lcnj;sbc;aiygc;iy</section>
+    <section class="description">
+      <div class="container">
+        <h1 class="description-text">How the app works</h1>
+      </div>
+    </section>
+    <section class="description-login">
+      <div class="container">
+        <div class="header-description">
+          <img class="description-login__img" src="images/Rectangle (1).png" alt="description-login__img">
```

Comment:
Wrap image in div so we can use it like column
For alt attribute use human understandable text
```suggestion
<div class="description-section__column description-section__left-column">
   <img class="description-section__column-img" src="images/Rectangle (1).png" alt="Login demo">
</div>
```